packages/providers/providers.yaml

```diff
@@ -2032,6 +2038,7 @@ codeclimate:
             method: GET
             endpoint: /v1/user
     docs: https://docs.nango.dev/integrations/all/codeclimate
+    docs_connect: https://docs.nango.dev/integrations/all/codeclimate/connect
     connection_config:
```

Comment:
You need to link the section to each field (using `doc_section`), you wrote some docs that nobody can read if you don't do that 😄